packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -540,16 +558,35 @@ function attemptExplicitHydrationTarget(
 
 export function queueExplicitHydrationTarget(target: Node): void {
   if (enableSelectiveHydration) {
-    const priority = getCurrentPriorityLevel();
+    const schedulerPriority = getCurrentPriorityLevel();
+    const updateLanePriority = higherLanePriority(
+      getCurrentUpdateLanePriority(),
+      DefaultLanePriority,
+    );
     const queuedTarget: QueuedHydrationTarget = {
       blockedOn: null,
       target: target,
-      priority: priority,
+      priority: schedulerPriority,
+      lanePriority: updateLanePriority,
```

Comment:
@sebmarkbage can you review the changes to hydration I made in this file? I'm adding the update lane priority with the goal of removing the scheduler priority in a future diff after we confirm update lane priority works as expected.